pkg/manifest/mutator.go

```
@@ -0,0 +1,342 @@
+	if containerName != "" {
+		return fmt.Errorf("container %s not found in deployment %s", containerName, deploymentName)
+	}
+
+	return fmt.Errorf("no containers found in deployment %s", deploymentName)
+}
+
+// ToYAML renders the mutated manifests back to YAML.
+func (m *Mutator) ToYAML() ([]byte, error) {
+	var buf bytes.Buffer
+	encoder := yaml.NewEncoder(&buf)
+	defer encoder.Close()
```

golangci-lint (2.11.4)

[error] Error return value of `encoder.Close` is not checked

(errcheck)